crates/hotshot/types/src/consensus.rs

```diff
@@ -1209,8 +1209,8 @@ impl<TYPES: NodeType> Consensus<TYPES> {
         is_epoch_transition(block_height, self.epoch_height)
     }
 
-    /// Returns true if our high QC is for the last block in the epoch
-    pub fn is_high_qc_for_last_block(&self) -> bool {
+    /// Returns true if our high QC is for one of the epoch transition blocks
+    pub fn is_high_qc_for_epoch_transition(&self) -> bool {
```

Comment:
Lets get rid of the leaf part, we can just get the height from the high_qc directly